test/test_tags.rb

```diff
@@ -14,6 +14,9 @@ def create_post(content, override = {}, converter_class = Jekyll::Converters::Ma
     if override['read_posts']
       site.posts.docs.concat(PostReader.new(site).read_posts(''))
     end
+    if override['read_collections']
```

Comment:
It would be preferable to make both these if's single line and stack them above the other.